.eslintrc.js

```diff
@@ -81,6 +81,7 @@ module.exports = {
     'quote-props': 'off',
     'space-before-function-paren': 'off',
     'react/jsx-no-bind': 'off', // Keep perf implications in mind, but was giving too many warnings and hurting readability
+    "react-hooks/exhaustive-deps": 'warn',
```

Comment:
@anicholls @NicholasBoll adding this only adds one warning for current code
```
canvas-kit-react/modules/modal/react/lib/ModalContent.tsx
  139:6  warning  React Hook React.useLayoutEffect has a missing dependency: 'handlerRef'. Either include it or remove the dependency array  react-hooks/exhaustive-deps
```
If y'all think we should add to the codebase I can autofix the warning